mlir/lib/Dialect/Rock/Transforms/GemmToGridwise.cpp

```diff
@@ -258,13 +259,16 @@ processGQA(ConversionPatternRewriter &rw, Location loc, Value queries,
     if (currentSeqLen)
       currentSeqLen =
           moveNumHeadsToSeqLenCurrSeqLen(rw, loc, currentSeqLen, numRepeats);
+    if (prefixOffset)
+      prefixOffset =
+          moveNumHeadsToSeqLenCurrSeqLen(rw, loc, prefixOffset, numRepeats);
```

Comment:
Should this be `if(currentSeqLen) else if(prefixOffset)` ? Can both be true at the same time ?